catalog/src/test/java/org/killbill/billing/catalog/io/TestVersionedCatalogLoader.java

```diff
@@ -37,6 +38,7 @@
 import org.killbill.billing.catalog.api.InvalidConfigException;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.weakref.jmx.internal.guava.io.Files;
```

Comment:
This is an internal (relocated) class, use `com.google.common.io.Files` directly.